<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homepage</class>
 <widget class="QDialog" name="homepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>741</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1121</width>
       <height>751</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
background-color: #0B2161;
border:none;
}
QLabel {
    color: black;
	background-color: rgb(255, 255, 255);
    font-size: 14px;
    font-weight: bold;
}

QComboBox {
    background-color: white;
    color: black;
    border: 1px solid blue;
    border-radius: 5px;
    padding: 2px;
}


GroupBox QComboBox:hover {
   
	background-color: rgb(175, 175, 177);
}
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="labelpost1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>331</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelpost2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>110</y>
        <width>341</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelpost3">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>110</y>
        <width>331</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="likeslabel1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>300</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
color:white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="likeslabel2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>300</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
color:white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="likeslabel3">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>290</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
color:white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>420</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>420</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox3">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>420</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
color: rgb(0, 0, 127);</string>
      </property>
      <property name="text">
       <string>Home Page </string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>520</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox5">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>520</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox6">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>520</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>300</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/redheart.png);
background-color: #0B2161;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>300</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/redheart.png);
background-color: #0B2161;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>300</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/redheart.png);
background-color: #0B2161;

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="heart1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>290</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
    background: transparent;
    border: none;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="heart2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>300</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
    background: transparent;
    border: none;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="heart3">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>290</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
    background: transparent;
    border: none;
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>380</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
color:white;</string>
      </property>
      <property name="text">
       <string>Comments:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>480</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
color:white;</string>
      </property>
      <property name="text">
       <string>Liked By:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>380</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
color:white;</string>
      </property>
      <property name="text">
       <string>Comments:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>480</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
color:white;</string>
      </property>
      <property name="text">
       <string>Liked By:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>380</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
color:white;</string>
      </property>
      <property name="text">
       <string>Comments:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>480</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
color:white;</string>
      </property>
      <property name="text">
       <string>Liked By:</string>
      </property>
     </widget>
     <zorder>label_4</zorder>
     <zorder>labelpost1</zorder>
     <zorder>labelpost2</zorder>
     <zorder>labelpost3</zorder>
     <zorder>likeslabel1</zorder>
     <zorder>likeslabel2</zorder>
     <zorder>likeslabel3</zorder>
     <zorder>comboBox1</zorder>
     <zorder>comboBox2</zorder>
     <zorder>comboBox3</zorder>
     <zorder>label</zorder>
     <zorder>comboBox4</zorder>
     <zorder>comboBox5</zorder>
     <zorder>comboBox6</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>heart1</zorder>
     <zorder>heart2</zorder>
     <zorder>heart3</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
     <zorder>label_7</zorder>
     <zorder>label_8</zorder>
     <zorder>label_9</zorder>
     <zorder>label_10</zorder>
    </widget>
    <widget class="QPushButton" name="next">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>700</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>next page</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>1131</width>
       <height>741</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #0B2161;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>751</width>
        <height>701</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
 background-color: white;
border: none;
border-radius:20px
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="prev">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>700</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>prev page</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>20</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/page1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>360</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">

border:none;
 border-radius: 60px; 

background-image: url(:/page4.jpg);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>520</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/page6.png);

border:none;
 border-radius: 60px; 
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>180</y>
        <width>120</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{
background-image: url(:/page5.jpg);
border:none;
border-radius: 60px; 
}
</string>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page1">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>70</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
    background-color:white;
    border: none;
}

QPushButton:hover {
    background-color: #f0f0f0; 
}

QPushButton:pressed {
    background-color: #ccc; 
}
</string>
      </property>
      <property name="text">
       <string>Following</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page2">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>220</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
    background-color:white;
    border: none;
}

QPushButton:hover {
    background-color: #f0f0f0; 
}

QPushButton:pressed {
    background-color: #ccc; 
}
</string>
      </property>
      <property name="text">
       <string>Following</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page3">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>390</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
    background-color:white;
    border: none;
}

QPushButton:hover {
    background-color: #f0f0f0; 
}

QPushButton:pressed {
    background-color: #ccc; 
}
</string>
      </property>
      <property name="text">
       <string>Following</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page4">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>560</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
    background-color:white;
    border: none;
}

QPushButton:hover {
    background-color: #f0f0f0; 
}

QPushButton:pressed {
    background-color: #ccc; 
}
</string>
      </property>
      <property name="text">
       <string>Following</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
